App/RadixWalletTests/KeychainActorTests.swift

```diff
@@ -0,0 +1,152 @@
+import Foundation
+import KeychainAccess
+import XCTest
+@_spi(KeychainInternal) import Prelude
+
+let authRandomKey: KeychainActor.Key = "authRandomDataKey"
+let noAuthRandomKey: KeychainActor.Key = "noAuthRandomDataKey"
+
+// MARK: - KeychainActorTests
+final class KeychainActorTests: XCTestCase {
+	let sut = KeychainActor.shared
+
+	func testNoAuth() async throws {
+		try await onceNoAuthTest()
+	}
+
+	func testAuth() async throws {
+		try await onceAuthTest()
+	}
+
+	func onceNoAuthTest() async throws {
+		try await sut.removeAllItems()
```

Comment:
I would call this in `setUp` and/or `tearDown` methods.

Reply:
sure, can change to that later - trying to avoid to much git admin today.